Modules/Axis/Sources/Axis/JSON/JSONMapParser.swift

```diff
@@ -81,6 +83,12 @@ public final class JSONMapParser : MapParser {
     public func parse(_ bytes: UnsafeBufferPointer<Byte>) throws -> Map? {
         let final = bytes.isEmpty
 
+        if !started {
+            started = true
+            // no bytes before we even started? empty json
+            if final { return .null }
```

Comment:
@Danappelxx I'd say here we should actually set `result = .null` and `return result` otherwise we end up with a parser that acts as if it is completed (returns a non-nil value) but actually can continue to parse bytes.